frontend/src/component/project/Project/ProjectInsights/ProjectInsights.tsx

```diff
@@ -15,42 +16,42 @@ const Grid = styled(Box)(({ theme }) => ({
     gridTemplateColumns: 'repeat(10, 1fr)',
 }));
 
-const Overview = styled(Box)(({ theme }) => ({
+const SpanFull = styled(Container)(() => ({
     gridColumn: '1 / -1',
 }));
 
-const HealthCard = styled(Container)(({ theme }) => ({
-    gridColumn: 'span 4',
-}));
-
-const ToggleTypesUsedCard = styled(Container)(({ theme }) => ({
+const Span2 = styled(Container)(() => ({
     gridColumn: 'span 2',
 }));
 
-const ProjectMembersCard = styled(Container)(({ theme }) => ({
-    gridColumn: 'span 2',
+const Span4 = styled(Container)(() => ({
```

Comment:
Can we give them some semantic meaning so that we don't need to rename variables when column width is changed?